nncf/common/sparsity/statistics.py

```diff
@@ -187,3 +187,41 @@ def to_str(self) -> str:
             f'Statistics of the RB-sparsity algorithm:\n{algorithm_string}'
         )
         return pretty_string
+
+class MovementSparsityStatistics(Statistics):
```

Comment:
unit test

Reply:
added in https://github.com/vuiseng9/nncf/pull/10/commits/6ced2896e7d9b836f609add9449fdaf5e60e622d

Reply:
@ljaljushkin is this the right place for this test?

Reply:
@vuiseng9, yes.